src/screens/UserPortal/Chat/Chat.tsx

```
@@ -118,51 +117,33 @@ export default function chat(): JSX.Element {
-        if (data && data.chatsByUser) {
+        if (data?.chatsByUser) {
           const filteredChats = orgId
-            ? data.chatsByUser.filter((chat: GroupChat | NewChatType) => {
-                if (isNewChatType(chat)) {
-                  return chat.organization?.id === orgId;
-                }
-                const legacy = chat as GroupChat;
-                return legacy.organization?._id === orgId;
-              })
+            ? data.chatsByUser.filter(
+              (chat: NewChatType) => chat.organization?.id === orgId,
```

ESLint

[error] Insert `··`

(prettier/prettier)